### DISCUSSION TITLE
Sleek management or Spambot2..?

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [community/community#138192] [Other Feature Feedback, Questions, & Ideas]
Up:
    https://github.com/orgs/community/discussions/137981
    https://github.com/orgs/community/discussions/70478#discussioncomment-10574263

    "Retiring this post as it is over 6 months old. If you have any questions, consider starting a new post."
    https://github.com/orgs/community/discussions?discussions_q=commenter:novium2+in:comments+%22Retiring+this+post+as%22

    Does validity really fade with time..?
    Are unnecessary spams, and bad aggregation: what really wanted?

    See also: [ https://github.com/MasterInQuestion/talk/discussions/31](https://github.com/MasterInQuestion/talk/discussions/31)

**Comment by novium2** [comment]
The posts are still there, if you want to see them. It's important to have people see the most recent and relevant announcements in this category.

**Comment by MasterInQuestion** [comment]
Thanks for your attention, anyway.

    However the truth remains:
    Those past are not necessarily past.
    And thoughtlessly starting new post would only result in bad aggregation.

    Maybe re-catalog alike to "Announcements (Past)", shall that matter?

**Reply by Marazali75** [reply]
U

**Reply by MasterInQuestion** [reply]
Is alike sensible?
    https://github.com/orgs/community/discussions/139935
    https://github.com/orgs/community/discussions/132506

    Separate posts for the same topic... How's the maintenance burden?

**Comment by MasterInQuestion** [comment]
This one is serious:
    https://github.com/orgs/community/discussions/54551#discussioncomment-10267836

    Loading the current [ [https://github.com/orgs/community/discussions/139005](https://github.com/orgs/community/discussions/139005) ] each time causes notorious bandwidth waste.
    The operator has to pay for the bandwidth cost.
    And I guess it matters for you. Or your bosses.

    Regardless. It appears GitHub has really lots of bandwidth at disposal...
    [ https://github.com/MasterInQuestion/attach/commit/80ff86187f5bbe807541b123993bdc77c9bd6cca#commitcomment-147535635](https://github.com/MasterInQuestion/attach/commit/80ff86187f5bbe807541b123993bdc77c9bd6cca#commitcomment-147535635)

    ----

    Unfortunately the mentioned problem hit the new commit page directly...
    https://github.com/orgs/community/discussions/139005#discussioncomment-10726668
    https://github.com/orgs/community/discussions/139005#discussioncomment-10694871
